### DISCUSSION TITLE
Finding LiDAR data

### DISCUSSION THREAD

**Discussion by rachelsadler** [cu-esiil-edu/stars-2024#35] [Q&A] [answered]
@eculler For our project I'm hoping to use LiDAR to show where debris flow has occurred in the years following the 2020 Grizzly Creek fire in Colorado (Garfield and Eagle Counties). I've searched through the [National Map](https://apps.nationalmap.gov/downloader/) and the [Colorado Hazard Mapping & Risk MAP Portal](https://coloradohazardmapping.com/lidarDownload) and can find lots of data from 2016. Any recommendations about where to find more recent data? If not, could I use satellite imagery instead?

**Comment by eculler** [comment, marked as answer]
Hi @rachelsadler, yeah that's always the challenge with using Lidar to look at these events, although usually our problem is more with getting data from before. I can ask around to see if anyone knows of some more recent LiDAR of the area. It can also be challenging to get LiDAR images to line up if they're not georeferenced, which could be a problem with unpublished data.

A lot of the geologists I know focus is on using synthetic aperture radar (SAR) to detect landslides. The Sentinel 1 mission is  a SAR sensor at 10m spatial resolution every 6 days, so depending on the size of landslides you're hoping to detect that can be a good choice. You can also cross-reference with rainstorms, whereas with the LiDAR you won't be able to pinpoint the dates. We can talk about some of the different methods used with SAR and what would be most appropriate if you want to go that route.  Are you just interested in the locations of debris flows? Or are you also looking for volume measurements?

**Reply by rachelsadler** [reply]
Hi @eculler, thank you! Sounds like using SAR is more aligned with the data I was looking for. I have precipitation data for the area so I will plan on using that to pinpoint dates. I am interested in the locations of debris flow. I'm curious to see the connections between where a debris flow event occurs and the slope, aspect, and vegetation of that area.

**Reply by eculler** [reply]
Hi @rachelsadler that sounds good. I was able to track down a high resolution DEM post-fire, from Worldview. I have it on my computer and can  send it to you. Would a OneDrive link work? It's a couple GB.

**Reply by rachelsadler** [reply]
Thank you, @eculler. That's great! OneDrive would work.
